src/devices_models/device_constructors/branch_constructor.jl

```diff
@@ -253,12 +259,17 @@ function construct_device!(
     sys::PSY.System,
     ::ModelConstructStage,
     device_model::DeviceModel{T, StaticBranchBounds},
-    network_model::NetworkModel{S},
-) where {T <: PSY.ACBranch, S <: StandardPTDFModel}
+    network_model::NetworkModel{<:StandardPTDFModel},
+) where {T <: PSY.ACBranch}
     devices =
         get_available_components(T, sys, get_attribute(device_model, "filter_function"))
     add_constraints!(container, NetworkFlowConstraint, devices, device_model, network_model)
-    branch_rate_bounds!(container, devices, device_model, S)
+    branch_rate_bounds!(
+        container,
+        devices,
+        device_model,
+        typeof(network_model).parameters[1],
```

Comment:
We have a getter function for this

```suggestion
        get_network_formulation(network_model)
```